build/lint.cloudbuild.yaml

```diff
@@ -21,4 +21,4 @@ tags:
 - 'lint'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.7'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.8'
```

Comment:
```suggestion
  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.10'
```

Reply:
I have used 1.10 already in the latest commit